pkg/virt-controller/watch/ovm.go

```diff
@@ -373,7 +371,7 @@ func setupStableFirmwareUUID(ovm *virtv1.OfflineVirtualMachine, vm *virtv1.Virtu
 	}
 
 	vm.Spec.Domain.Firmware.UUID = types.UID(uuid.NewSHA1(firmwareUUIDns, []byte(vm.ObjectMeta.Name)).String())
-	logger.Infof("Setting stabile UUID '%s' (was '%s')", vm.Spec.Domain.Firmware.UUID, existingUUID)
+	logger.V(4).Infof("Setting stable UUID '%s' (was '%s')", vm.Spec.Domain.Firmware.UUID, existingUUID)
```

Comment:
This log still does not make sense. Why would you want to print the "old" uuid, when we explicitly never override any given uuids?